### ISSUE TITLE
Harvester failing to link dataset organization to already existing organization 

### ISSUE THREAD

**Issue by ogsletcax** [cioos-siooc/ckanext-cioos_harvest#1]
The organisation https://cnckan.cioos.ca/organization/fisheries-and-oceans-canada was created when harvesting pacific datasets

The dataset https://cnckan.cioos.ca/dataset/c2c3e218-7704-48fa-8c18-327a5a88e28e has been harvested from atlantic but cannot be linked to https://cnckan.cioos.ca/organization/fisheries-and-oceans-canada because the organization guid it is using is different

**Comment by fostermh** [comment]
Interesting and good point. As I understand it however we have decided to change the way we use organization and harvesting.

In your example you would have set created remote organizations in the harvesting config to true for both pacific and atlantic.

I believe we agreed to use organization as who the data is harvested from and rely on "responsible organization" to indicate that multiple entries have input into a given dataset. As such, at the national level, you should not create any organizations automatically and assign all datasets to the RA you are harvesting from. This can be done by simply assigned the harvest to the correct organization.

**Comment by ogsletcax** [comment]
ok so maybe we should remove

  "remote_groups": "create",
  "remote_orgs": "create",

from the harvest configuration in the documentation?

**Comment by fostermh** [comment]
Good point and yes I agree. Feel free to do that or I can when I’m next in the office.

**Comment by ogsletcax** [comment]
cool i opened the pull request: https://github.com/cioos-siooc/ckan/pull/15/commits/59bc14f595758bcf1bc837c60264478a2f16407e
